associacao-nacional-de-pesquisa-e-ensino-em-transportes.csl

```diff
@@ -425,7 +425,7 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-use-last="true" entry-spacing="0" hanging-indent="true">
+  <bibliography hanging-indent="false" entry-spacing="12" subsequent-author-substitute="______" subsequent-author-substitute-rule="complete-all">
```

Comment:
`entry-spacing="12"` can't possible be right. Entry spacing describes the number of rows between bibliography entries and there clearly shouldn't be 12. Most likely 1 is correct?

Reply:
Here I use  entry-spacing="12pt"`. But if I use `pt`, do not pass in the automatic test.

Reply:
@hildogjr, sorry for the late reply, but see http://docs.citationstyles.org/en/stable/specification.html#whitespace for additional explanation of how the "entry-spacing" attribute can be used in CSL 1.0.1 styles. It may only be set to an integer (whole) number to specify how many lines separate bibliographic entries.

If you need more precise spacing you might be able to do this by manipulating the final output, e.g. with CSS in HTML or document styles in Word.